src/mantine-core/src/Rating/Rating.styles.ts

```diff
@@ -0,0 +1,92 @@
+import { createStyles, MantineSize } from '@mantine/core';
```

Comment:
You cannot import anything from `@mantine/core` package, it will break the build. Import from `@mantine/styles` instead.